org.sixtyninestudios.kublit.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.sixtyninestudios.kublit
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Kublit.sh
+finish-args: 
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+  - --device=dri
+modules:
+  - name: Kublit
+    buildsystem: simple
+    build-commands:
+      - install -D Kublit.sh /app/bin/Kublit.sh
+      - install -D kubliticon.png /app/share/icons/kubliticon.png
+      - install -D org.sixtyninestudios.kublit.desktop /app/share/applications/org.sixtyninestudios.kublit.desktop
+      - install -D org.sixtyninestudios.kublit.metainfo.xml /app/share/metainfo/org.sixtyninestudios.kublit.metainfo.xml
+      - install -D KublitRelease.x86_64 /app/lib/Kublit/KublitRelease.x86_64
```

Comment:
so you are creating an aarch64 flatpak with x86_64 binaries.

Reply:
I guess

Reply:
I'll clarify. This should be fixed.

Reply:
Wdym, the game is made in godot.

Only the executable is x86

Reply:
```
/app/bin/Kublit.sh: line 2: /app/lib/Kublit/KublitRelease.x86_64: cannot execute binary file: Exec format error
```

Reply:
Tru now

Reply:
no it's not,

1. the chmod will fail because the files system is read-only
2. what I didn't mention is that it was on aarch64. Which was expected since you ignored by review comments and : https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson

Note that if you built from source (we have support from that) this wouldn't even be a problem.

Reply:
You need to add an arm template to godot and export it for that platform.